### PR TITLE
Updated General Wording and Fixed Typos

### PR DESCRIPTION
**Pull request by DanielleQuinn**
Improved general narrative, clarified some language, standardized format of code chunks, and fixed typos.

I will open a couple issues noted re: wording / code details that I wasn't able to address myself.

### CODE REVIEW

**Comment by DanielleQuinn** [comment]
Sorry, should specify - changes were made to lesson 06-Using a relational database with R

File: _episodes/06-Using a relational database with R.md [hunk 1]
```diff
@@ -20,78 +20,84 @@ keypoints:
...
 
-This can be done in a single line of code
+Connect to the SQLite database specified by `dbfile`, above, using the `dbConnect` function.
 
 ~~~
 mydb <- dbConnect(dbDriver("SQLite"), dbfile)
 ~~~
 
-'mydb' represents the connection to the database. It will be specified everytime we need to access the database.
+Here, `mydb` represents the connection to the database. It will be specified everytime we need to access the database.
```
Comment (ErinBecker) [inline _episodes/06-Using a relational database with R.md]: every time should be two words

File: _episodes/06-Using a relational database with R.md [hunk 2]
```diff
@@ -114,42 +120,39 @@ SQL_query <- paste("SELECT q.value,",
...
 
 > ## Exercise
 >
-> What happens if you send invalid SQL?
+> What happens if you send invalid SQL syntax?
 > 
 > > ## Solution
 > > 
-> > An error message is reurned from SQLite, R is just the conduit, it cannot check the SQL syntax.
+> > An error message is reurned from SQLite.
```
Comment (ErinBecker) [inline _episodes/06-Using a relational database with R.md]: reurned should be returned

**Comment by ErinBecker** [comment]
LGTM - just a few typo corrections and then this can be merged.

**Comment by ErinBecker** [comment]
@DanielleQuinn "Oh! I saw those typos but because the entire sentence is a bit of a mystery I added it as an issue that someone with more knowledge of the topic might be able to fix - so I didn't touch the multiple typos in the sentence itself. I can, though!"

Merging with the remaining typos. These can be handled at a later point when that sentence is cleaned up.

**Pull request merged**